tests/README.rst

```diff
@@ -0,0 +1,89 @@
+.. FogLAMP test scripts describes how to FogLAMP scripted tests are organised and how to write the scripted tests
+
+.. |br| raw:: html
+
+   <br />
+
+.. Links
+
+.. Links in new tabs
+
+.. |pytest docs| raw:: html
+
+   <a href="https://docs.pytest.org/en/latest/contents.html" target="_blank">pytest</a>
+
+.. |pytest decorators| raw:: html
+
+   <a href="https://docs.pytest.org/en/latest/mark.html" target="_blank">pytest</a>
+
+.. _Unit: unit\\README.rst
+.. _Integration: integration\\README.rst
+.. _System: system\\README.rst
+.. _here: ..\\README.rst
+
+.. =============================================
+
+********************
+FogLAMP Test Scripts
+********************
+
+FogLAMP scripted tests are classified into three categories:
+
+- `Unit`_ - Tests that checks the expected output of a code block.
+- `Integration`_ - Tests that checks the integration of different FogLAMP units that work as a single component.
+- `System`_ - Tests that checks the end to end flows in FogLAMP
+
+
+Running FogLAMP scripted tests
+==============================
+
+Test Prerequisites
+------------------
+
+Follow the instructions mentioned `here`_  to install and run FogLAMP on your machine.
+You can test FogLAMP from your development environment or after installing FogLAMP.
+
+In order to execute the scripted tests, run the following command from FOGLAMP_ROOT
+::
+   pip3 install -r python/requirements-test.txt --user
```

Comment:
This does not run any tests, it is getting the dependencies the tests need - please reword as it stand it makes it sound as if the above command executes tests.